doc/manpages/mapping.asciidoc

```diff
@@ -0,0 +1,52 @@
+KAKOUNE(1)
+==========
+
+NAME
+----
+mapping - a
+
+Description
+-----------
+Creating and removing shortcuts boils down to the following functions,
+respectively:
+
+---------------------------------------
+map [flags] <scope> <mode> <key> <keys>
+unmap <scope> <mode> <key> [<expected>]
+---------------------------------------
+
+The *map* command makes *key* behave as if the *keys* sequence was typed.
+
+*mode* dictates in what context the mapping will be available:
+
+	*insert*::
+		insert mode (default: 'i')
+	*normal*::
+		normal, commands mode
+	*prompt*::
+		prompts, inputs (e.g. '/')
+	*menu*::
+		multiple items dropdown menus (e.g. ':')
```

Comment:
Hum, not really, the menu mode is a specific one, which is actually pretty rare, AFAIR only ctags.kak and spell.kak use it. In menu mode we dont have anything else we can do than select a menu item.

That makes me think it might be time to rework that menu mode, but its another story.